packages/amplify-provider-awscloudformation/src/aws-utils/aws-ssm.ts

```diff
@@ -1,6 +1,7 @@
 import { $TSContext, $TSObject } from 'amplify-cli-core';
 import { AwsSecrets, loadConfiguration } from '../configuration-manager';
 import aws from './aws.js';
+import * as AWS from 'aws-sdk';
```

Comment:
Is this required for the TS namespace issue? Can we add a comment in here stating the reason please? (since AWS var is not used anywhere in this file specifically)

Reply:
we are using to initialize the client 

```
readonly client: AWS.SSM;
```